juristische-schulung.csl

```diff
@@ -0,0 +1,291 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="note" default-locale="de-DE">
+  <info>
+    <title>Juristische Schulung (German)</title>
+    <title-short>JuS</title-short>
+    <id>http://www.zotero.org/styles/juristische-schulung</id>
+    <link href="http://www.zotero.org/styles/juristische-schulung" rel="self"/>
+    <link href="https://www.zotero.org/styles/juristische-zitierweise" rel="template"/>
```

Comment:
@adam3smith, https://github.com/citation-style-language/styles/blob/master/juristische-schulung.csl still has to point to "neue-juristische-wochenschrift" instead, right?